doc/security/index.rst

```diff
@@ -0,0 +1,160 @@
+| 2020-04-23 | `CVE-2020-1760`_  | 6.1 Medium  | potential XSS attack                                    |
++------------+-------------------+-------------+---------------------------------------------------------+
+| 2020-04-13 | `CVE-2020-1759`_  | 6.8 Medium  | Cephx nonce reuse in secure mode                        |
++------------+-------------------+-------------+---------------------------------------------------------+
+| 2020-02-07 | `CVE-2020-1700`_  | 6.5 Medium  | RGW disconnects leak sockets, can DoS                   |
++------------+-------------------+-------------+---------------------------------------------------------+
+| 2020-04-21 | `CVE-2020-1699`_  | 7.5 High    | Dashboard path traversal flaw, info dislosure           |
++------------+-------------------+-------------+---------------------------------------------------------+
+| 2019-11-08 | `CVE-2019-10222`_ | 7.5 High    | Invalid HTTP headers could crash RGW                    |
++------------+-------------------+-------------+---------------------------------------------------------+
+| 2019-03-27 | `CVE-2019-3821`_  | 7.5 High    | RGW file descriptors could be exhausted                 |
++------------+-------------------+-------------+---------------------------------------------------------+
+| 2019-01-28 | `CVE-2018-16889`_ | 7.5 High    | encryption keys logged in plaintext                     |
++------------+-------------------+-------------+---------------------------------------------------------+
+| 2019-01-15 | `CVE-2018-16846`_ | 6.5 Medium  | authenticated RGW users can cause DoS                   |
++------------+-------------------+-------------+---------------------------------------------------------+
+| 2019-01-15 | `CVE-2018-14662`_ | 5.7 Medium  | read-only users could steal dm-crypt keys               |
++------------+-------------------+-------------+---------------------------------------------------------+
+| 2018-07-10 | `CVE-2018-10861`_ | 8.1 High    | authenticated user can create/delete pools              |
++------------+-------------------+-------------+---------------------------------------------------------+
+| 2018-03-19 | `CVE-2018-7262`_  | 7.5 High    | malformed headers can cause RGW DoS                     |
++------------+-------------------+-------------+---------------------------------------------------------+
+| 2018-07-10 | `CVE-2018-1129`_  | 6.5 Medium  | network MITM can tamper with messages                   |
++------------+-------------------+-------------+---------------------------------------------------------+
+| 2018-07-10 | `CVE-2018-1128`_  | 7.5 High    | Cephx replay attack vulnerability                       |
++------------+-------------------+-------------+---------------------------------------------------------+
+| 2017-12-12 | `CVE-2017-12155`_ | 6.3 Medium  | ceph.client.openstack.keyring world readable            |
```

Comment:
CVE-2017-12155 affects OpenStack product, issue was fixed in "openstack-tripleo-heat-templates/puppet-tripleo", did we ever fixed it in ceph?